std/socket.d

```diff
@@ -1663,6 +1663,22 @@ public:
     }
 
     /**
+     * Compares with another InternetAddress of same type for equality
+     * Returns: true if the InternetAddresses share the same address and
+     * port number.
+     * Examples:
+     * --------------
+     * InternetAddress addr1,addr2;
+     * if (addr1 == addr2) { }
```

Comment:
Uhm.. won't this example segfault due to a `null this`? I forgot whether this was ever fixed (or ever will be).